https://sources.debian.org/src/budgie-desktop/10.5.1+git20200715-2/debian/patches/desktop-override.patch/
Description: Reinstate desktop overrides
 Upstream no longer wants to use desktop overrides.
 Reinstate these for Debian & Ubuntu to ensure
 clean separation from Budgie and GNOME desktops
Author: David Mohammed <fossfreedom@ubuntu.com>
Forwarded: not-needed
Last-Update: 2019-03-09


Index: budgie-desktop-10.5~git20190308/src/wm/20_solus-project.budgie.wm.gschema.override
===================================================================
--- budgie-desktop-10.5~git20190308.orig/src/wm/20_solus-project.budgie.wm.gschema.override
+++ budgie-desktop-10.5~git20190308/src/wm/20_solus-project.budgie.wm.gschema.override
@@ -1,7 +1,7 @@
-[org.gnome.mutter]
+[org.gnome.mutter:Budgie]
 edge-tiling = true
 attach-modal-dialogs = true

-[org.gnome.desktop.wm.preferences]
+[org.gnome.desktop.wm.preferences:Budgie]
 button-layout = 'appmenu:minimize,maximize,close'
